README.md

```diff
@@ -50,3 +50,137 @@ Follow the steps below to run Export. To ensure .py files run when double-clicke
+* The 'date' refers to the date of the scenario analysis. 
+  * Hurricane, Tsunami HPR do not yet have Analysis Date
+  * Needs to be in 'YYYY-MM-DD' format
+* You can change 'modifiedInventory' to true or false. Default is false.
+* You can change the 'source'. Max 100 characters (may be larger).
+* Meta can contain any info.
+  * This is a json object or python dictionary, one level deep.
+  * Need to use double quotes and separate each item by spaces i.e. {"firstname":"Patty","nickname":"Pat"}
+* event must match an id in Event.csv
+* Required fields: id, name, hazard, analysisType, date, source, modifiedInventory, event
+
+**Renaming Study Regions and Scenarios for HLL:**
+
+To rename the Study Region you can open the 'Event.csv' and modify the value in the 'name' field.
+
+To rename the Scenario you can open the 'Analysis.csv' and modify the value in the 'name' field.
+
+#### Download.csv ####
+
+![Event.csv](Images/batchDownloadCSV.png "Event.csv")
+
+* You can add a url to the 'link' field for a row and HLL button for that item will open the link instead of the file.
+* Category and subcategory can be changed to group and sort the files for an analysis.
+* Probabilistic analysis will have an Annual return period and it will not have a hazardBoundaryPoly.zip.
+* Meta can contain any info.
+  * This is a json object or python dictionary, one level deep.
+  * Need to use double quotes and separate each item by spaces i.e. {"firstname":"Patty","nickname":"Pat"}
+* analysis must match an id in Analysis.csv
+* Required fields: id, category, subcategory, name, icon, analysis
+
+## Requirements
+
+Same as Hazus Export.
+
+## To Use
+
+Follow the steps below to run Batch Export. To ensure .py files run when double-clicked, right-click the .py file and go to Properties. Under the "General" tab next to "Opens With", make sure "python.exe" is selected. If not, click "Change" and select "python.exe" from your Python installation directory.
+
+It's recommended that if you want to rerun an HPR that you should delete or rename the previous run's output folder for that HPR. 
+
+**1. Place your .hpr files into the batch_input folder**
+
+![batch input folder](Images/batchInputFolder.png "batch input folder")
+
+**1b. Adjust the desired outputs by modifying the values in the 'hazpy/batch_export.py'**
+Look near the very bottom of the script for these entries:
+
+    #USER DEFINED VALUES
+    hprDir = Path.absolute(Path(r'./batch_input'))      #The directory containing hpr files
+    outDir = Path.absolute(Path(r'./batch_output'))     #The directory for the output files
+    _outCsv = 1         #Export csv files: 1 to export or 0 to skip
+    _outShapefile = 1   #Export shapefiles: 1 to export or 0 to skip
+    _outReport = 1      #Export report pdf files: 1 to export or 0 to skip
+    _outJson = 1        #Export json files: 1 to export or 0 to skip
+
+If you wanted to export just the reports it would need to look like this:
```

Comment:
minor grammar nit: change to '**_If you want to export only the reports, then it would need to look like this_**:

This is the correct grammar when using 'then' as a conjunction, i.e.: if/then statements.